### ISSUE TITLE
onFullScreen Event

### ISSUE THREAD

**Issue by lord-xeon** [xiaolin/react-image-gallery#93]
Can you add a hook for onFullScreen change?

**Comment by xiaolin** [comment]
Sure.

**Comment by lord-xeon** [comment]
I made the change I was looking for on my fork, I'm getting authorization errors trying to upload a pull request.

All I need is something simple:


```
//In fullScreen()
if(this.props.onFullScreen){
      this.props.onFullScreen(true);
}

//in exitFullScreen()
if(this.props.onFullScreen){
      this.props.onFullScreen(false);
}
```

**Comment by lord-xeon** [comment]
I'm trying so track some stuff with Google Analytics, all I need is a boolean for enter/exit

**Comment by xiaolin** [comment]
Got it, I'll return the `fullscreenElement` then.  
```
  _handleScreenChange() {
    const fullScreenElement = document.fullscreenElement ||
      document.msFullscreenElement ||
      document.mozFullScreenElement ||
      document.webkitFullscreenElement;

    if (this.props.onScreenChange) {
      this.props.onScreenChange(fullScreenElement);
    }

    this.setState({isFullscreen: !!fullScreenElement});
  }
``

**Comment by lord-xeon** [comment]
Awesome, thanks for the quick response!

This is the perfect little package that covers everything I was looking for btwz.

**Comment by xiaolin** [comment]
np, glad you find it useful! Released the callback in v0.7.1
